plugin/exchange.vim

```diff
@@ -325,6 +325,7 @@ function! s:get_setting(setting, default)
 endfunction
 
 highlight default link ExchangeRegion IncSearch
+execute 'highlight default link _exchange_region' (s:enable_highlighting ? 'ExchangeRegion' : 'None')
```

Comment:
Since `s:enable_highlighting` is defaulted to `1` at startup, let's do this more plainly:

```suggestion
highlight default link _exchange_region ExchangeRegion
```

Reply:
done